### DISCUSSION TITLE
Reactions plugin: Add config option for limiting issues *specifically*, not comments

### DISCUSSION THREAD

**Discussion by spenserblack** [lowlighter/metrics#334] [Ideas]
Per the discussion in #333, issues take priority over comments when being limited. This behavior can be somewhat confusing for users that expect both recent issues *and* issue comments to be mixed together in their reactions metric. To allow issue comments to also be counted if the number of issues exceeds the limit, it could be helpful to add an option like `plugin_reactions_limit_issues` as suggested in this comment: https://github.com/lowlighter/metrics/issues/333#issuecomment-849893709. This could allow issues and their comments to be limited separately.

**Comment by lowlighter** [comment]
Shipped in 7f08abb 🎉 !
